src/libserver/cfg_file.h

```diff
@@ -364,7 +364,6 @@ struct rspamd_config {
 	gboolean check_text_attachements;               /**< check text attachements as text					*/
 	gboolean check_all_filters;                     /**< check all filters									*/
 	gboolean allow_raw_input;                       /**< scan messages with invalid mime					*/
-	gboolean disable_hyperscan;                     /**< disable hyperscan usage							*/
```

Comment:
This should not be removed - it is an used feature to disable HS from the configuration.